electron-app/src/components/settings/PremiumSettings.vue

```diff
@@ -2,16 +2,25 @@
   <v-row class="premium-settings">
     <v-col>
       <v-card>
-        <v-card-title>Premium Settings</v-card-title>
+        <v-card-title>Rotki Premium</v-card-title>
         <v-card-text>
+          <p>
+            Rotki Premium is an optional subscription service to gain access to
+            analytics, graphs, and unlock many additional features. For more
+            information on what is available visit the
+            <base-external-link href="https://rotki.com/products/">
```

Comment:
`ElectronIntrop` (probably bad choice for a name we should change it) should have the `premiumUrl` property why not use that instead of hardcoding it? It would be better to have the links in one place in geenral

Reply:
I'm hard-coding it because if there is no $interop then it won't render the href properly (i.e. if the app is being run from the browser but no electron).

Reply:
@isidorosp yes that is why I mentioned that the name is a bad choice. This class is available in both platforms any suggestions for a suitable name?

Reply:
Sorry I didn't realize that interop is available even when we're not in electron. In that case we need to amend the code in https://github.com/rotki/rotki/blob/3d2ae3604bb576576c6a1f38a80caaffb1f3f5b4/electron-app/src/components/base/BaseExternalLink.vue because it just checks if $interop exist in order to determine whether to render a normal <a href="..."> link or to use an onclick event (if we're in electron).

Reply:
Changed it to `$interop.isPackaged` like you had originally suggested (I confirmed it works in dev as well).